concordium-contracts-common/src/lib.rs

```diff
@@ -89,7 +89,11 @@ mod traits;
 mod impls;
 pub mod constants;
 pub mod schema;
+#[cfg(feature = "derive-serde")]
+pub mod schema_json;
 mod types;
+#[cfg(feature = "derive-serde")]
+pub mod versioned_schema_helpers;
```

Comment:
It is hard to use a library that has exports scattered like this.

The `schema_json` should just be part of the `schema` (feature gated of course) module since there is one function there. The `write_bytes_from_json_schema_type` should instead be a method on the `Type`, i.e, 

```
impl Type {
    pub fn write_bytes_from_json_schema(&self, json: &serde_json::Value, out: ...) -> Result<(), JSONConversionError> {
        ....
   }
}